apps/etl-func/docs/tokenize.puml

```diff
@@ -0,0 +1,29 @@
+@startuml TokenizeFiscalCode
+
+participant caller
```

Comment:
imho we can remove this actor

Reply:
I think that having a caller helps. Maybe I can use an actor instead of a partecipant?